client/src/components/RecipeSearch.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react"
+import SearchedRecipeTile from "./SearchedRecipeTile"
+
+const RecipeSearch = (props) => {
+  const [formData, setFormData] = useState({
+    searchQuery: ""
+  })
+
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const [searchResults, setSearchResults] = useState(undefined)
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+    try {
+      const response = await fetch("/api/v1/recipe-search", {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(formData)
+      })
+      if(response.ok) {
+        const body = await response.json()
+        setSearchResults(body.searchResults)
+      }
+    } catch(err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
```

Comment:
so this implementation is honestly fine. If you wanted to follow best conventions though, you would make this a GET fetch request, and add the search query to the end of the path as query params
```
const response = await fetch(`/api/v1/recipe-search?query=${formData.searchQuery}`
```